nixos/modules/services/networking/i2pd.nix

```diff
@@ -115,18 +144,18 @@ let
       default = null;
     };
     i2cp.leaseSetAuthType = mkOption {
-      type = with types; nullOr int;
-      description = lib.mdDoc "Authentication type for encrypted LeaseSet. 0 - no authentication, 1 - DH, 2 - PSK.";
+      type = with types; nullOr (enum (builtins.attrNames typeMaps.i2cp.leaseSetAuthType));
+      description = lib.mdDoc "Authentication type for encrypted LeaseSet.";
       default = null;
     };
     i2cp.leaseSetClient.dh.nnn = mkOption {
       type = with types; nullOr str;
-      description = lib.mdDoc "Client name:client's public DH in base64, for authentication type 1.";
+      description = lib.mdDoc "Client name:client's public DH in base64.";
```

Comment:
Don't remove clarifications from descriptions (unless of course they're invalid)